### ISSUE TITLE
[now dev] failed to validate now.json when using "includeFiles"

### ISSUE THREAD

**Issue by haowen737** [vercel/vercel#3408]
Hi,
I'm trying to implement _Including Additional File_ as [docs](https://zeit.co/docs/configuration#project/functions) says.

After I config my `now.json` file with `functions` property 

```
{
  "functions": {
    "api/test.js": {
      "includeFiles": "templates/**"
    }
  }
}
```

`now dev` run failed with `includeFiles` in `now.json`

```
> Now CLI 16.6.2 dev (beta) — https://zeit.co/feedback/dev
> Error! Invalid `functions` property: ['api/test.js'] should NOT have additional properties
```

**Comment by haowen737** [comment]
I have read the code from the `master` branch,

https://github.com/zeit/now/blob/b69f1761439b92e058aa7b9c30be0475073bcf38/packages/now-cli/src/util/dev/validate.ts#L47-L62

It seems both `includeFiles` and `excludeFiles` are missing in functionsSchema, which is called by function `validateNowConfig` 

What do you think 🤔?

**Comment by styfle** [comment]
Thanks! This was fixed in #3414 and available on the canary channel!

```
npm i -g now@canary
```
